apps/auction2/components/auction_grid/index.styl

```diff
@@ -0,0 +1,2 @@
+.auction-grid
```

Comment:
Be sure to be diligent about namespacing by containing app or component folder (`auction2-grid` > `auction-grid`. I had to search + replace the partner2 app recently to [fix some style conflicts](https://github.com/artsy/force/pull/738).